app/models/auth_token.rb

```diff
@@ -1,18 +1,60 @@
 # frozen_string_literal: true
 
-class AuthToken < ActiveRecord::Base
-  belongs_to :access_code
+class AuthToken < ActiveModelSerializers::Model
```

Comment:
`ActiveRecord` is backed by a database table (we used to store the token in the database for checking later), but now we're not saving them. However since `AuthTokenSerializer` needs something to serialize, `ActiveModelSerializers::Model` adds the necessary methods for serialization.